packages/near-membrane-base/src/environment.ts

```diff
@@ -176,17 +175,8 @@ export class VirtualEnvironment {
 
     lazyRemap(target: ProxyTarget, ownKeys: (string | symbol)[]) {
         const targetPointer = this.blueGetTransferableValue(target) as Pointer;
-        const enumerableKeys = ObjectKeys(target);
         const args: any[] = [targetPointer];
-        for (let i = 0, { length } = ownKeys; i < length; i += 1) {
-            const key = ownKeys[i];
-            const isEnumerable =
-                typeof key === 'symbol'
-                    ? ReflectApply(ObjectProtoPropertyIsEnumerable, target, [key])
-                    : ReflectApply(ArrayProtoIncludes, enumerableKeys, [key]);
-
-            ReflectApply(ArrayProtoPush, args, [key, isEnumerable]);
-        }
```

Comment:
Because lazy remapping is based on an object shape these keys already exist and already have their enumerability set. We can skip it and simply let the natural enumerability (true or false) be respected.